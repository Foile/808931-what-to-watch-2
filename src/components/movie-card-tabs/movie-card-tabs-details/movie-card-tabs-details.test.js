import React from "react";
import renderer from "react-test-renderer";
import MovieCardTabsDetails from "./movie-card-tabs-details";

it(`MovieCardTabsReviews correctly renders after relaunch`, () => {
  const tree = renderer
  .create(<MovieCardTabsDetails />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});
