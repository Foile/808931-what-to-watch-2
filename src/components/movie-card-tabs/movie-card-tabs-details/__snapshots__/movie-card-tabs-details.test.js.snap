// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCardTabsReviews correctly renders after relaunch 1`] = `
<div
  className="movie-card__text movie-card__row"
>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Director
      </strong>
      <span
        className="movie-card__details-value"
      />
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Starring
      </strong>
      <span
        className="movie-card__details-value"
      />
    </p>
  </div>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Run Time
      </strong>
      <span
        className="movie-card__details-value"
      >
        0h 0m
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Genre
      </strong>
      <span
        className="movie-card__details-value"
      />
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Released
      </strong>
      <span
        className="movie-card__details-value"
      />
    </p>
  </div>
</div>
`;
